tools/cellxgene_census_builder/pyproject.toml

```diff
@@ -36,7 +36,7 @@ dependencies= [
     #    https://github.com/TileDB-Inc/TileDB/blob/dev/format_spec/FORMAT_SPEC.md
     "tiledbsoma==1.9.3",
     "cellxgene-census==1.12.0",
-    "cellxgene-ontology-guide==0.6.1",
+    "cellxgene-ontology-guide==1.*",
```

Comment:
We should have this pinned - upgrading a dependency that can affect the data components of the builder should be tested every time.

Reply:
```suggestion
    "cellxgene-ontology-guide==1.0.0",
```